src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -4867,7 +4868,9 @@ def test_set_up_azure_service_mesh(self):
         ground_truth_mc_2 = self.models.ManagedCluster(
             location="test_location",
             service_mesh_profile=self.models.ServiceMeshProfile(
-                mode="Istio", istio=self.models.IstioServiceMesh()
+                mode="Istio", istio=self.models.IstioServiceMesh(
+                    revisions=["asm-1-88"]
```

Comment:
do we also need a test case here for if revisions is empty? or is that implicit?

Reply:
thanks for pointing this out. Added a UT for the same.